base/Plain_netCDF_Time.F90

```diff
@@ -908,4 +908,60 @@ subroutine split_string_by_seperator (string_in, length_mx, seperator_in, &
     return
   end subroutine split_string_by_seperator
 
+
+  
+  
+  subroutine scan_write_between_line1_line2_flush_Left (ur, uw, L1, L2)
+    character (len=*), intent(in) :: L1, L2
+    integer, intent(in) :: ur, uw
+
+    integer :: i, j, k
+    character (len=150) :: line
+
+    rewind(ur)
+    ios=0
+    j=0
+    k=0
+    do while (ios==0)
+       read (ur, '(a150)', iostat = ios, err = 300) line
```

Comment:
I think this procedure could be simplified a fair bit.   You generally don't need both `iostat` and `err`.  Indeed, you are not actually using iostat here and could eliminate the variable `ios` entirely.

Also, `j` appears to be more of a flag (`logical`) than an integer.

But on the whole, if it is working for you, we can save this for the MAPL3 rewrite.

Reply:
I am not sure about the real difference between iostat and err, although this works for me. So I will keep   it for others to have a systematic improvement. Thanks.